.github/workflows/main.yml

```diff
@@ -0,0 +1,47 @@
+# Sync miner docs from https://github.com/tellor-io/telliot/tree/master/docs in miner-documentation folder in the current repo
```

Comment:
We should rename the Miner folder to Telliot and will refactor the docs to explain that one of the tasks that Telliot does is mining.